support-frontend/assets/pages/[countryGroupId]/router.tsx

```diff
@@ -52,7 +53,7 @@ const LandingPage = lazy(() => {
 });
 const StudentLandingPage = lazy(() => {
 	return import(
-		/* webpackChunkName: "StudentLandingPage" */ './studentLandingPage'
+		/* webpackChunkName: "StudentLandingPage" */ './student/StudentLandingPage'
```

Comment:
Ok you want to move this into its own `student` directory like adLite. Was not sure whether to do this as the rest of the pages (landing/checkout/thankyou/router) are a level up.  However, keeps things neat and tidy, I will move my faq and ts&cs components into the folder.